build.gradle

```diff
@@ -354,7 +354,7 @@ configure(project(':monitor')) {
 
 
 configure(project(':pricenode')) {
-    apply plugin: "org.springframework.boot"
+    mainClassName = 'bisq.price.Main'
```

Comment:
I am not familiar with Spring. Is org.springframework.boot not needed amymore?

Reply:
To be honest, I am not familiar with it either. But the only way I found to eliminate the error with unable to load main class when running the start script was to not apply the plugin. However, there is now an issue with trying to run the jar. I will continue to investigate.